extensions/amp-ad/0.1/test/test-amp-ad-3p-impl.js

```diff
@@ -210,14 +210,52 @@ function tests(name) {
           resizable: '',
         }, 'https://schema.org').then(element => {
           const impl = element.implementation_;
-          const attemptChangeSizeSpy = sandbox.spy(impl, 'attemptChangeSize');
-          impl.apiHandler_.updateSize_(217, 114);
-          expect(attemptChangeSizeSpy.callCount).to.equal(1);
-          expect(attemptChangeSizeSpy.firstCall.args[0]).to.equal(217);
-          expect(attemptChangeSizeSpy.firstCall.args[1]).to.equal(114);
+          const attemptChangeSize = sandbox.stub(impl, 'attemptChangeSize',
```

Comment:
The description is confusing indeed. I changed it.

`ampHandler_` don't have unit test now. All tests are in `test-amp-ad-3p-impl` I agree we should move some tests to `ampHandler` but that involves more tests other than these two. I would prefer to do it in another PR.